src/panels/lovelace/special-rows/hui-call-service-row.ts

```diff
@@ -1,83 +1,34 @@
-import {
-  html,
-  LitElement,
-  TemplateResult,
-  customElement,
-  property,
-  css,
-  CSSResult,
-} from "lit-element";
-import "@material/mwc-button";
+import { customElement } from "lit-element";
 
-import "../../../components/ha-icon";
-
-import { callService } from "../common/call-service";
-import { LovelaceRow, CallServiceConfig } from "../entity-rows/types";
-import { HomeAssistant } from "../../../types";
+import { ButtonRowConfig } from "../entity-rows/types";
+import { HuiButtonRow } from "./hui-button-row";
 
 @customElement("hui-call-service-row")
-class HuiCallServiceRow extends LitElement implements LovelaceRow {
-  public hass?: HomeAssistant;
-
-  @property() private _config?: CallServiceConfig;
+export class HuiCallServiceRow extends HuiButtonRow {
+  public setConfig(config: ButtonRowConfig): void {
```

Comment:
```suggestion
  public setConfig(config: CallServiceConfig): void {
```

Reply:
Can't change the signature of setConfig

Reply:
Yeah already "fixed" it...

Reply:
I think the best way would be

```ts
const buttonRowConfig = {...config, type: 'button'};

super.setConfig(buttonRowConfig);
```

Similar to how the Sensor card does for entity card extending